.github/workflows/ci.yml

```diff
@@ -758,13 +815,15 @@ jobs:
         target:
           - platform: "linux"
             arch: "x86_64"
-            docker_image: "gcr.io/iree-oss/base@sha256:f26a5aa5f8d3705c6b80c71d04fafb360861f1907bdd1b1f5f19480b6192664e"
+            docker_image: "gcr.io/iree-oss/base-bleeding-edge@sha256:10a94f7edf0df2f3894038e0e434a93ea353460ed72240c226548025690a79e5"
             # Builds tools on the host and assumes the builder is Linux x86_64.
             build_script: "./build_tools/cmake/build_runtime.sh"
+            tracy_platform_arch: "linux-x86_64"
```

Comment:
It's a little bit tricky because tracy capture tool doesn't always run on the target benchmark machine (e.g. mobile phone benchmarks run tracy on RPis). So I have a separate job to build tracy capture tool and fetch the proper built binary here.